<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="inputPow_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input power INC, dBm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="com_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>COM Port</string>
    </property>
    <widget class="QPushButton" name="com_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Open port</string>
     </property>
    </widget>
    <widget class="QComboBox" name="com_combobox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="comstate_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Closed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="echo_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Echo</string>
     </property>
    </widget>
    <widget class="QLabel" name="service_message">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Empty</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="com_refresh_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="UM78_supply_group">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Power Amplifier Supply Control</string>
    </property>
    <widget class="QCheckBox" name="plus_5_UM78_checkbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+5V</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="minus_5_UM78_checkbox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>-5V</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="plus_3_UM78_checkbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+3.3V</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="plus_2_5_UM78_checkbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+2.5V</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UM78_supply_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="UM78_Vd_checkbox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Vd</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="outputREFpow_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>450</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output power REF, dBm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="input_lcdNumber">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="preAmp_groupbox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Preamplifier Supply Control</string>
    </property>
    <widget class="QCheckBox" name="plus_5_checkbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+5V</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="minus_5_checkbox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>-5V</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="plus_3_checkbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+3.3V</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="plus_2_5_checkbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+2.5V</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="HMC_checkbox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>RF_+5V</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supply_ctrl_send_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="plus_28_checkbox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+28V</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Vd_checkbox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Vd</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="outputREF_lcdNumber">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>450</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="UM78_gate_group">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>PowAmp Gate/Drain Control</string>
    </property>
    <widget class="QSlider" name="UM78_gate_slider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2500</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="sliderPosition">
      <number>2500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="UM78_gate_line_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="UM78_gate_push_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Setup Gate Voltage</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UM78_setup_gate_voltage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Power ON</string>
     </property>
    </widget>
    <widget class="QLabel" name="UM78_drain_current_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Drain Current</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="UM78_read_drain_current">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Read Drain Current</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Attenuator Control</string>
    </property>
    <widget class="QSlider" name="att_slider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>63</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="att_line_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="att_ctrl_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QComboBox" name="att_address_combo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="outputINCpow_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output power INC, dBm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>PreAmp Gate/Drain Control</string>
    </property>
    <widget class="QSlider" name="gate_slider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2500</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="sliderPosition">
      <number>2500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="gate_line_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="gate_push_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Setup Gate Voltage</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setup_gate_voltage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Power ON</string>
     </property>
    </widget>
    <widget class="QLabel" name="drain_current_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Drain Current</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Read Drain Current</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="strobCtrl_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Strobe control</string>
    </property>
    <widget class="QPushButton" name="strobPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pulseWid_lineedit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dutyCycle_lineedit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="pulseWid_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pulse width, us</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="duty_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Duty cycle, %</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="strobe_checkbox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Modulation on/off</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="stopPowButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="outputINC_lcdNumber">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="comparatorGroupBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Comparator Control</string>
    </property>
    <widget class="QSlider" name="compPR_slider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="compPR_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="compPR_lineedit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="compPA_slider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="compPA_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="compPA_lineedit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCalibrate"/>
   </widget>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCalibrate">
   <property name="text">
    <string>Calibrate</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
